src/ecdsa/ecdh.py

```diff
@@ -109,57 +130,78 @@ def load_private_key(self, private_key):
 
     def load_private_key_bytes(self, private_key):
         """
-        Load private key from plain bytes string.
-        Uses current curve and checks if key length corresponds to
-        the current curve.
+        Load private key from byte string.
```

Comment:
empty line missing after summary line

Reply:
also, I think referencing SigningKey method, so that the format of the byte string is defined, would be a good idea